README.md

```diff
@@ -50,17 +50,25 @@ func (myHandler) HandleImageMessage(message whatsapp.ImageMessage) {
 	fmt.Println(message)
 }
 
+func (myHandler) HandleDocumentMessage(message whatsapp.VideoMessage) {
```

Comment:
Thank you for contributing to the project! 👍 
another small mistake was made: HandleDocumentMessage takes a DocumentMessage as parameter and not VideoMessage.

Reply:
> After that small fix I will directly merge the branch. Sorry that I did not spot that yesterday alreasy

No problem, my bad, I did not pay attention at the details